kolibri/core/assets/src/views/textbox/index.vue

```diff
@@ -76,14 +77,23 @@
         currentText: this.value,
       };
     },
+    watch: {
+      value: 'setCurrentText',
+    },
```

Comment:
This was [already added](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/textbox/index.vue#L88) in `develop` so it should be removed from here.

After merge, currently throws:

```
/Users/d/Projects/learning_equality/repos/kolibri/kolibri/core/assets/src/views/textbox/index.vue
      82:7  error  Duplicate key 'value'  no-dupe-keys
```